lms/static/sass/certificates/_certificate-template-01.scss

```diff
@@ -170,9 +170,27 @@
 
 .a--accomplishment-design-01__certificate-info-print-only {
   overflow: visible;
+  color: #fff;
+
+  .info-upper {
+    @include display(flex);
+    @include align-items(center);
+    margin-bottom: 1em;
+  }
+
+  .info-right {
+    margin-left: auto;
+  }
+
+  a {
+    color: #fff;
+    font-weight: bolder;
+  }
 
   @media screen {
-    display: none;
+    //display: none;
```

Comment:
Please remove this. I don't see why we'd keep it as a comment.

Reply:
Ugh, forgot about that. Fixed :)